app/views/trip_families/new.html.erb

```diff
@@ -0,0 +1,3 @@
+<p class="title"><%= t(".title") %></p>
+<p class="title">Still to allocate</p>
```

Comment:
Translation

Reply:
Actually I don't need the title any more because we removed the "still to allocate" method and partial. Removed with fixup in sha `e2a4c4e`